mobile/src/main/AndroidManifest.xml

```diff
@@ -95,5 +95,20 @@
                 <action android:name="android.intent.action.MY_PACKAGE_REPLACED" />
             </intent-filter>
         </receiver>
+
+        <receiver
+            android:name=".background.AlarmChangedReceiver"
+            android:enabled="true"
+            android:exported="true">
+            <intent-filter>
+                <action android:name="android.app.action.NEXT_ALARM_CLOCK_CHANGED" />
+            </intent-filter>
+        </receiver>
+        <receiver android:name=".background.BackgroundTaskRetryBroadcastReceiver" />
```

Comment:
Could maybe unify those two into one receiver?

Reply:
Both receivers might be doing different things in the future, e.g. the LocaleChange one could recreate the cloud notification channels, so I want to keep them as two receivers. Also the BackgroundTaskRetryBroadcastReceiver isn't exported.

Reply:
Exported is a fair point, but LocaleChangedReceiver and AlarmChangedReceiver could be unified, couldn't they? I guess I just don't like the introduction of 3 (!) manifest registered receivers for such a small feature.

Reply:
I merged both exported one